homeassistant/components/matter/sensor.py

```diff
@@ -1181,6 +1237,31 @@ def _update_from_device(self) -> None:
         # don't discover this entry if the supported state list is empty
         secondary_value_is_not=[],
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SENSOR,
+        entity_description=MatterSensorEntityDescription(
+            key="RvcOperationalStateOperationalError",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            translation_key="operational_error",
+            device_class=SensorDeviceClass.ENUM,
```

Comment:
Ah, one more small thing: Can we reorder the keys to fit the first error sensor schema? (which is similar to other schemas now)
```suggestion
            translation_key="operational_error",
            device_class=SensorDeviceClass.ENUM,
            entity_category=EntityCategory.DIAGNOSTIC,
```